src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -136,7 +136,7 @@ const BioAndChat = (props) => {
           labelLeft={
             isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
           }
-          styleLabelLeft={{color: isAnonimityEnabled ? colors.white : '#648ABF'}}
+          styleLabelLeft={{color: isAnonimityEnabled ? COLORS.white : COLORS.signed_secondary}}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color for `styleLabelLeft` has been updated to use the new `COLORS` object. The hardcoded color value `'#648ABF'` has been replaced with `COLORS.signed_secondary`.
```diff
- styleLabelLeft={{color: isAnonimityEnabled ? colors.white : '#648ABF'}}
+ styleLabelLeft={{color: isAnonimityEnabled ? COLORS.white : COLORS.signed_secondary}}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->